packages/editor-ui/src/views/TestDefinition/TestDefinitionEditView.vue

```diff
@@ -83,12 +104,69 @@ function hasIssues(key: string) {
 	return fieldsIssues.value.some((issue) => issue.field === key);
 }
 
-watch(() => state.value, debounce(onSaveTest, { debounceTime: 400 }), { deep: true });
+async function onDeleteMetric(deletedMetric: Partial<TestMetricRecord>) {
+	if (deletedMetric.id) {
+		await deleteMetric(deletedMetric.id, testId.value);
+	}
+}
+
+async function handleCreateTag(tagName: string) {
+	try {
+		const newTag = await tagsStore.create(tagName);
+		return newTag;
+	} catch (error) {
+		toast.showError(error, 'Error', error.message);
+		throw error;
+	}
+}
+
+async function openPinningModal() {
+	uiStore.openModal(NODE_PINNING_MODAL_KEY);
+}
+
+async function runTest() {
+	await testDefinitionStore.startTestRun(testId.value);
+	await testDefinitionStore.fetchTestRuns(testId.value);
+}
+
+const runs = computed(() =>
+	Object.values(testDefinitionStore.testRunsById ?? {}).filter(
+		(run) => run.testDefinitionId === testId.value,
+	),
+);
+
+async function onDeleteRuns(runs: TestRunRecord[]) {
+	await Promise.all(
+		runs.map(async (run) => {
+			await testDefinitionStore.deleteTestRun({ testDefinitionId: testId.value, runId: run.id });
+		}),
+	);
+}
+
+// Debounced watchers for auto-saving
+watch(
+	() => state.value.metrics,
+	debounce(async () => await updateMetrics(testId.value), { debounceTime: 400 }),
+	{ deep: true },
+);
+
+watch(
+	() => [
+		state.value.description,
+		state.value.name,
+		state.value.tags,
+		state.value.evaluationWorkflow,
+		state.value.mockedNodes,
+	],
+	debounce(onSaveTest, { debounceTime: 400 }),
+	{ deep: true },
+);
 </script>
 
 <template>
 	<div :class="$style.container">
-		<div :class="$style.content">
+		<div :class="$style.formContent">
+			<!-- Name -->
 			<EvaluationHeader
```

Comment:
The side panel is cut off while scrolling. 

<img width="515" alt="image" src="https://github.com/user-attachments/assets/b7015d11-6cb5-4872-9ae9-2c401de7bcaa" />